internal/csi-addons/networkfence/fencing_test.go

```diff
@@ -54,3 +54,43 @@ func TestGetIPRange(t *testing.T) {
 		})
 	}
 }
+
+func TestFetchIP(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		clientInfo  string
+		expectedIP  string
+		expectedErr bool
+	}{
+		{
+			clientInfo:  "client.4305 172.21.9.34:0/422650892",
```

Comment:
I guess this also needs to support IPv6 addresses? You probably should test if that works too.

Reply:
Currently it works for IPv4, will add support for IPv6 in a separate new PR? can open an issue for it as a tracker.

Reply:
Ideally add support in this PR. IPv6 should not be treated differently from IPv4.
If you really want to address it in a different PR, create an issue and leave a link here.

Reply:
reported at #4192